website/content/docs/concepts/variables.mdx

```diff
@@ -7,24 +7,24 @@ description: Learn about the Nomad Variables feature
 # Nomad Variables
 
 Most Nomad workloads need access to config values or secrets. Nomad has a
-`template` block to provide such configuration to tasks, but has left the role
-of storing that configuration to external services such as [HashiCorp Consul]
-and [HashiCorp Vault].
+`template` block to provide such configuration to tasks, but prior to Nomad 1.4
+has left the role of storing that configuration to external services such as
+[HashiCorp Consul] and [HashiCorp Vault].
 
-Nomad Variables provide the option to securely store configuration at file-like
-paths directly in Nomad's state store. The contents of these secrets are
-encrypted and replicated between servers via raft. Access to secrets is
-controlled by ACL policies, and tasks have implicit ACL policies that allow them
-to access their own secrets. You can create, read, update, or delete secrets via
-the command line, the Nomad API, or in the Nomad web UI.
+Nomad Variables provide the option to store configuration at file-like paths
+directly in Nomad's state store. The contents of these variables are encrypted
+and replicated between servers via raft. Access to variables is controlled by
+ACL policies, and tasks have implicit ACL policies that allow them to access
+their own variables. You can create, read, update, or delete variables via the
+command line, the Nomad API, or in the Nomad web UI.
 
 Note that the Variables feature is intended for small pieces of configuration
 data needed by workloads. Because writing to the Nomad state store uses
 resources needed by Nomad, it is not well-suited for large or fast-changing
 data. For example, do not store batch job results as Variables - these should be
 stored in an external database. Variables are also not intended to be a full
-replacement for HashiCorp Vault. If you need powerful options like dynamic
-secrets or transit encryption, continue using Vault.
+replacement for HashiCorp Vault. Unlike Vault, Nomad stores the root encryption
+key on the servers. See [Key Management] for details.
```

Comment:
The links work fine without the extra `[]` but it syntax highlights in most markdown modes in editors if you have it in there, which is handy:

```suggestion
key on the servers. See [Key Management][] for details.
```